src/py/crankshaft/crankshaft/query_runner.py

```diff
@@ -0,0 +1,43 @@
+"""class for fetching data"""
+import plpy
+
+
+class QueryRunner:
+    def get_markov(self, query):
```

Comment:
Ok, makes sense. And `__init__` would construct the query.. Which would clean up the code in other functions like `Moran.local_stat` :)